src/Training.AirBnb.Clone.Backend/Backend Project/Domain/Services/NotificationsServices/EmailTemplateService.cs

```diff
@@ -13,45 +13,45 @@ public EmailTemplateService(IDataContext dataContext)
     {
         _dataContext = dataContext;
     }
-
+    
     public async ValueTask<EmailTemplate> CreateAsync(EmailTemplate emailTemplate, bool saveChanges = true, CancellationToken cancellationToken = default)
     {
 
-        if (!ValidationIsNull(emailTemplate))
+        if (!ValidationToNull(emailTemplate))
             throw new EmailTemplateValidationToNull("This a member of these emailTemplate null");
-
-        if (ValidationExits(emailTemplate))
-            throw new EmailTemplateAlreadyExists("This emailTemplate already exists");
-
+        
+        if(ValidationExits(emailTemplate))
+            throw new  EmailTemplateAlreadyExists("This emailTemplate already exists");
+        
         await _dataContext.EmailTemplates.AddAsync(emailTemplate, cancellationToken);
-
-        if (saveChanges)
-            await _dataContext.EmailTemplates.SaveChangesAsync();
+        
```

Comment:
save changesda cancallation berish kerak

Reply:
buni to'g'riladim fewature/#63